src/main/java/com/pinterest/secor/common/SecorConfig.java

```diff
@@ -237,6 +237,10 @@ public String getUploaderClass() {
         return getString("secor.upload.class", "com.pinterest.secor.uploader.Uploader");
     }
 
+    public String getUploaderRetries() {
+        return getString("secor.upload.retries", "5");
```

Comment:
Can you add this property in secor.common.properties with default value.  We prefer setting the default value in properties file instead of java class.

Is default value 3 better than 5?